core/controllers/base_test.py

```diff
@@ -42,10 +42,23 @@
 
 class BaseHandlerTest(test_utils.GenericTestBase):
 
+    TEST_LEARNER_EMAIL = "test.learner@example.com"
+    TEST_LEARNER_USERNAME = "testlearneruser"
+    TEST_CREATOR_EMAIL = "test.creator@example.com"
+    TEST_CREATOR_USERNAME = "testcreatoruser"
+
     def setUp(self):
         super(BaseHandlerTest, self).setUp()
         self.signup('user@example.com', 'user')
 
+        # create a learner user to test learner redirect behavior
```

Comment:
> Please pay attention to punctuation, capitalization, etc., 
Should I pay attention to punctuation ,capitalization etc. in comments or in 45-48 ? If in line 45-48 what specs should I follow?